CHANGELOG.md

```diff
@@ -345,3 +345,13 @@ Note: gradle version 0 and 1 are note supported.
 * Initial release with support for Cognito UserPools, Cognito Identity, and API key based authentication.
 * Optimistic updates allow the cache to be updated before a server response is received (i.e. slow network or offline)
 * Offline mutation allows mutations to be queued while client is offline until online again.
+# Release 3.1.4
```

Comment:
This is getting added to the bottom of the file, but should be added to the top instead.